packages/localStorage/index.tsx

```diff
@@ -0,0 +1,45 @@
+export function useSafeLocalStorage(): Storage {
+   const safeLocalStorage = {} as Storage;
```

Comment:
I think the type coercion could mask potential type errors.

```suggestion
   const safeLocalStorage = {};
```